src/quafu/circuits/quantum_circuit.py

```diff
@@ -53,6 +54,10 @@ def parameterized_gates(self):
             self._parameterized_gates = [g for g in self.gates if g.paras is not None]
         return self._parameterized_gates
 
+    @property
+    def num(self):
+        return sum([len(qreg) for qreg in self.qregs])
```

Comment:
Yes, qc.reg() will be added later, together with some kind of way to index to "physical qubits". So that there will be full supports for qasm, annotation and fancy-style indexing.